cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -134,6 +136,16 @@ internal class SpConsentLibImpl(
     }
 
     private fun localLoadMessage(authId: String?, pubData: JSONObject?, cmpViewId: Int?) {
+
+        if (context.isInternetConnected().not()) {
+            spClient.onError(
```

Comment:
@bohdan-go-wombat what do you think about moving these checks to `spClient`? This way we could check it on before _every_ call?

Reply:
if we want to do so, we should move it either to the `NetworkClient` or to the `chech { ... }` method

Reply:
oh, my bad, I meant to say `NetworkClient`

Reply:
@andresilveirah I think that makes sense, I was just not really confident about passing context instance that much, because that might cause memory leaks

but let me try that out